test/authenticated.js

```diff
@@ -1051,3 +1051,24 @@ describe('Plaid.Client - Longtail Institutions', function() {
   });
 
 });
+
+describe('Plaid.Client - All Institutions', function() {
+  var client =
+    new Plaid.Client('test_id', 'test_secret', Plaid.environments.tartan);
+
+  // The /institutions/all endpoint requires additional permissions
+  // to access. Accessing it with the sandbox client ID and secret returns
+  // an error, which we test here.
```

Comment:
This comment is identical to the comment on line 1038–1040. This test is essentially a copy & paste of that test. Was simply attempting to maintain equal test coverage for this new method as was there for the long tail method.